libraries/Filter/LowPassFilter2p.cpp

```diff
@@ -83,6 +87,10 @@ float LowPassFilter2p<T>::get_sample_freq(void) const {
 
 template <class T>
 T LowPassFilter2p<T>::apply(const T &sample) {
+    if (!is_positive(_params.cutoff_freq)) {
```

Comment:
Applying this to https://github.com/ardupilot/ardupilot/blob/master/libraries/Filter/LowPassFilter.cpp#L104 might make that clearer, too.

Reply:
LowPassFilter.cpp already handles this case